lib/Extension/SourceCodeFilesystemExtra/SourceCodeFilestem/Application/ClassSearch.php

```diff
@@ -36,6 +36,7 @@ public function __construct(FilesystemRegistry $filesystemRegistry, FileToClass
     public function classSearch(string $filesystemName, string $name)
     {
         $filesystem = $this->filesystemRegistry->get($filesystemName);
+        $name = str_replace('\\', '/', $name);
```

Comment:
This class isn't under test currently, and one of the "big" upcoming tasks would be to provide a dedicated service for finding classes and other things (e.g. functions), and this service would replace this code anyway...